api/server/router/network/network_routes.go

```diff
@@ -8,11 +8,11 @@ import (
 
 	"golang.org/x/net/context"
 
-	"github.com/docker/docker/api/server/httputils"
-	"github.com/docker/docker/api/types"
-	"github.com/docker/docker/api/types/filters"
-	"github.com/docker/docker/api/types/network"
-	"github.com/docker/docker/api/types/versions"
+	"github.com/moby/moby/api/server/httputils"
+	"github.com/moby/moby/api/types"
+	"github.com/moby/moby/api/types/filters"
+	"github.com/moby/moby/api/types/network"
+	"github.com/moby/moby/api/types/versions"
```

Comment:
Gogland did this automatically for me (I think it runs `goimports`) but some imports may need to be re-sorted after the rename

Reply:
`go fmt`'d everything on the last push.

Reply:
Awesome, thanks!

Reply:
Yes, we need to run goimports on all the files that got modified.

Reply:
`gofmt` should be sufficient in this case.